docs/products/kafka/howto/kafka-tiered-storage-get-started.rst

```diff
@@ -0,0 +1,37 @@
+
+Get started with tiered storage for Aiven for Apache Kafka® 
+====================================================================
+
+Aiven for Apache Kafka®'s tiered storage expands storage beyond local disks. It stores frequently accessed data on faster tiers and less active data on cost-effective, slower tiers, ensuring both performance and cost efficiency are optimized.
+
+For an in-depth understanding of tiered storage, how it works, and its benefits, see `Tiered Storage in Aiven for Apache Kafka®`.
```

Comment:
I guess it's the name of a section? I can't find it.

Reply:
This content is related to the concept topics in the PR - https://github.com/aiven/devportal/pull/2142. I'm unable to create direct links at the moment, as they would lead to broken links and prevent me from merging the content into the main branch.